src/main/java/net/mcreator/ui/init/TiledImageCache.java

```diff
@@ -55,12 +55,16 @@ public class TiledImageCache {
 	public static ImageIcon bucket;
 	public static ImageIcon bucketMask;
 
+	public static ImageIcon spawnEggBase;
+	public static ImageIcon spawnEggDots;
+
 	public static void loadAndTileImages() {
 		try {
 			TiledImageUtils plantGrowthTile = new TiledImageUtils(UIRES.get("growthtile"), 128, 215);
 			TiledImageUtils modTabTile = new TiledImageUtils(UIRES.get("taboverlaytile"), 64, 64);
 			TiledImageUtils workspaceIcons = new TiledImageUtils(UIRES.get("wrktile"), 45, 45);
 			TiledImageUtils bucketIcons = new TiledImageUtils(UIRES.get("fluidbucket"), 32, 32);
+			TiledImageUtils spawnEggIcons = new TiledImageUtils(UIRES.get("spawnEgg"), 32, 32);

```

Comment:
```suggestion
			TiledImageUtils spawnEggIcons = new TiledImageUtils(UIRES.get("spawnegg"), 32, 32);
```

Reply:
Java is case-sensitive :D

Reply:
It is not Java, but Linux FS that makes file names case-sensitive. Windows ignores them, that is why it passed on my PC.

Good thing we have tests and a nice catch!